src/main/java/com/google/api/codegen/gapic/GapicGeneratorFactory.java

```diff
@@ -184,6 +185,9 @@
               newCsharpGenerator.apply(
                   CSharpBasicPackageTransformer.forUnitTests(unitTestPathMapper)));
         }
+        if (devSamples) {
+          generators.add(newCsharpGenerator.apply(new CSharpStandaloneSampleTransformer()));
```

Comment:
Can we call this `CSharpGapicSamplesTransformer` for consistency with the other languages, or does this conflict  with an existing class?

Reply:
Yeah, CSharp has a `CSharpGapicSnippetsTransformer`. It doesn't conflict with `CSharpGapicSamplesTransformer`, but does cause some confusion (Snippets? Samples?) IMO. So I decided to rename the transformer for standalone samples.